Makefile

```diff
@@ -65,7 +66,7 @@ SYSTEM_TEST_TARGET?=./tests/system
 PYTEST_OPTIONS?=--timeout=90 --durations=20 --junit-xml=build/TEST-system.xml
 
 .PHONY: check-full
-check-full: update check golint staticcheck
+check-full: update check golint staticcheck lint
```

Comment:
This is going to break developers' workflows.
Currently, check-full will succeed as long as all code is OK. With this change, the code needs to be checked in as well. The developer then needs to check in the code before they can tell if it it's valid, which is backwards.

Reply:
I added the dependency on `fmt` to `check`. Is that correct?

Reply:
I would prefer if "check" was non-modifying, but I think this is OK for now as it's fixing a hole we have at the moment. Ultimately I'd like to do something like the Go Agent does, checking if formatting is required without actually doing it: https://github.com/elastic/apm-agent-go/blob/master/scripts/check_goimports.sh)